scripts/anvil_tdr_ingest/clone_anvil_workspace.py

```diff
@@ -5,8 +5,9 @@
 
 import argparse
 import json
-import pandas as pd
-import requests
+# not used some commenting out for now
```

Comment:
please delete these commented out lines if they're not used!

Reply:
Sorry forgot about that cleanup

Reply:
cleaned up now.